PROVESFlightControllerReference/test/int/bridge_uplink_test.py

```
@@ -0,0 +1,471 @@
+    timeout = int(os.environ.get("UPLINK_TIMEOUT", "120"))
+    with open(src, "rb") as f:
+        digest = hashlib.md5(f.read()).hexdigest()
+    print(
+        f"\n[uplink] src={src} dest={dest} size={os.path.getsize(src)} md5={digest} timeout={timeout}s"
+    )
+    import time as _t
+
+    print(f"UPSTART {_t.time():.3f}", flush=True)
+    try:
+        fprime_test_api.uplink_file_and_await_completion(src, dest, timeout=timeout)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)